nucypher/blockchain/eth/deployers.py

```diff
@@ -315,7 +315,7 @@ def upgrade(self, existing_secret_plaintext: bytes, new_secret_hash: bytes):
 
         # Raise if not all-systems-go
         self.check_deployment_readiness()
-        origin_args = {'from': self.deployer_address, 'gas': 5000000}  # TODO
+        origin_args = {'from': self.deployer_address, 'gas': 5000000}  # TODO: Gas management
```

Comment:
Does TODO this need an issue?

Reply:
<https://github.com/nucypher/nucypher/issues/842>